build-logic-commons/code-quality-rules/src/main/resources/checkstyle/required-header.txt

```diff
@@ -12,4 +12,4 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
\ No newline at end of file
+ */
```

Comment:
> I left a few comments. I think the PR got too huge now. Can you start with a PR for the java/groovy/Kotlin sources and then we'd see if we want to do follow up PRs for other text files?

So be it, I will close this, move those to a new one, and apply any comment from here which applies there :+1:

Reply:
I think this might be GH not adding the emoji at the and, as from the terminal, it's obvious:

```
 git --no-pager diff origin/master -- build-logic-commons/code-quality-rules/src/main/resources/checkstyle/required-header.txt
```

```diff
diff --git a/build-logic-commons/code-quality-rules/src/main/resources/checkstyle/required-header.txt b/build-logic-commons/code-quality-rules/src/main/resources/checkstyle/required-header.txt
index 719aef866cf..b935e1c7a3f 100644
--- a/build-logic-commons/code-quality-rules/src/main/resources/checkstyle/required-header.txt
+++ b/build-logic-commons/code-quality-rules/src/main/resources/checkstyle/required-header.txt
@@ -12,4 +12,4 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
\ No newline at end of file
+ */
```

Reply:
Probably this is some weird edge-case in the processing on their side.